index.html

```diff
@@ -84,14 +84,151 @@ <h1>Hey there.<span class="span__br"><br /></span> I’m Mateo.
+      <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
+
+        <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
+
+        <div class="project_wrapper"></div>
+        <article class="project_card">
+          <div class="project_card_wrapper">
+            <h4 class="project_card_title">Multi-Post Stories Gain+Glory</h4>
+            <ul class="project_card_tags">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+
+            <button class="project_card_button" type="button">See project</button>
+              
+          </div>
+        </article>
     </div>
+  </section>
+
+  <!-----About Me--------->
+
+  <section id="about">
+    <div class="about_me">
+      <div class="about_container">
+        <h2>About me</h2>
+
+        <p class="about_description">
+          I can help you build a product , feature or website
+           Look through some of my work <span class="span__br2"><br /></span> 
+           and experience! If you like what you see and have a project 
+           you need coded, don’t hesitate to contact me.
+        </p>
+
+        <button class="about_button" type="button">
+          <a href="#">Get my resume</a>
+        </button>
+      </div>
+
+      <div class="about_wrapper">
+        <article class="about_me_container languages">
+          <div class="about_languages">
+            <div class="about_icon">
+              <img
+                src="./assets/icons/LanguageIcon.svg"
+                />
+            </div>
+
+            <h3 class="about_title">Languages</h3>
+
+            <ul class="about_cardtags">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+        </article>
+
+        <article class="about_me_container frameworks">
+          <div class="about_languages">
+            <div classs="about_icon">
```

Comment:
- [ ] Your technology categories such as `languages, frameworks and skills` should be wrapped in `ul` and `li` tags. It is important to make use of semantic list item tags to display the list of technologies. :+1: